locale/en/blog/release/v0.10.45.md

```diff
@@ -24,7 +24,7 @@ Commits:
 * [[`8df4b0914c`](https://github.com/nodejs/node/commit/8df4b0914c)] - **deps**: separate sha256/sha512-x86_64.pl for openssl (Shigeki Ohtsu) [joyent/node#25654](https://github.com/joyent/node/pull/25654)
 * [[`11eefefb17`](https://github.com/nodejs/node/commit/11eefefb17)] - **deps**: copy all openssl header files to include dir (Shigeki Ohtsu) [#6553](https://github.com/nodejs/node/pull/6553)
 * [[`61ccc27b54`](https://github.com/nodejs/node/commit/61ccc27b54)] - **deps**: upgrade openssl sources to 1.0.1t (Shigeki Ohtsu) [#6553](https://github.com/nodejs/node/pull/6553)
-* [[`aa02438274`](https://github.com/nodejs/node/commit/aa02438274)] - **openssl**: fix keypress requirement in apps on win32 (Shigeki Ohtsu) [joyent/node#25654](https://github.com/joyent/node/
+* [[`aa02438274`](https://github.com/nodejs/node/commit/aa02438274)] - **openssl**: fix keypress requirement in apps on win32 (Shigeki Ohtsu) [joyent/node#25654](https://github.com/joyent/node/pull/6553)
```

Comment:
I don't think this is right. It should be https://github.com/joyent/node/pull/25654

Reply:
I'm not sure if the trees were merged when the repos were rejoined. Are you suggesting pointed to the IO.js repo for these?

Reply:
Sorry, I meant https://github.com/joyent/node/pull/25654

Reply:
Damn, GitHub changes the link automatically :P

Reply:
This `https://github.com/joyent/node/pull/25654`